libs/eo/transfers/src/lib/eo-transfers-form.component.ts

```diff
@@ -172,6 +181,15 @@ interface EoTransfersForm {
           An 8-digit TIN/CVR number is required
         </watt-error>
       </watt-form-field>
+      <watt-form-field>
+        <watt-label>Wallet Deposit Endpoint</watt-label>
```

Comment:
Is this inline with the UI from Pernille? And is it possible to see the design for this made by Pernille?

Reply:
There is actually a link to a figma board, on the zenhub issue. I will adjust the design, so it fits with that. thanks for pointing it out

Reply:
Adjusted the design now. Can we resolve this?